SPECS/perl-JSON/perl-JSON.spec

```diff
@@ -0,0 +1,304 @@
+%global __provides_exclude %{?__provides_exclude:%__provides_exclude|}perl\\(JSON::(Backend::PP|backportPP::Boolean|Boolean|PP|PP::IncrParser)\\)
+%global __requires_exclude %{?__requires_exclude:%__requires_exclude|}perl\\(JSON::(backportPP|backportPP::Boolean)\\)
+Summary:        Parse and convert to JSON (JavaScript Object Notation)
+Name:           perl-JSON
+Version:        4.02
+Release:        5%{?dist}
+License:        GPL+ OR Artistic
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://metacpan.org/release/JSON
+#Source0:       https://cpan.metacpan.org/modules/by-module/JSON/JSON-%{version}.tar.gz
+Source0:        %{name}-%{version}.tar.gz
+BuildRequires:  coreutils
+BuildRequires:  findutils
+BuildRequires:  make
+BuildRequires:  perl
+BuildRequires:  perl(Carp)
+BuildRequires:  perl(Data::Dumper)
+BuildRequires:  perl(Encode)
+BuildRequires:  perl(Exporter)
+BuildRequires:  perl(ExtUtils::MakeMaker)
+BuildRequires:  perl(File::Spec)
+BuildRequires:  perl(Math::BigFloat)
+BuildRequires:  perl(Math::BigInt)
+BuildRequires:  perl(Scalar::Util)
+BuildRequires:  perl(Test)
+BuildRequires:  perl(Test::More)
+BuildRequires:  perl(base)
+BuildRequires:  perl(constant)
+BuildRequires:  perl(lib)
+#Requires:       perl(:MODULE_COMPAT_%(eval "`%{__perl} -V:version`"; echo $version))
```

Comment:
If this is being commented out, it should be replaced with a `Requires: perl`. Note this only applies to `1.0` and `1.0-dev`.